web/src/components/tasks/business-formation/business/BusinessStep.test.tsx

```diff
@@ -464,7 +468,7 @@ describe("Formation - BusinessStep", () => {
   });
 
   describe("Will Practice Law", () => {
-    for (const legalStructureId of ["c-corporation", "s-corporation"]) {
+    for (const legalStructureId of corporateLegalStructures) {
```

Comment:
[pebble] why aren't we using the it.each pattern here as well?

Reply:
Just wan't my focus to correct, all the for loops to it.each loops in general. I think that is better suited for a follow up refactor.